client/Graphics.cpp

```diff
@@ -443,4 +444,15 @@ void Graphics::initializeImageLists()
 		addImageListEntry(spell->id, "SPELLBON", spell->iconScenarioBonus);
 		addImageListEntry(spell->id, "SPELLSCR", spell->iconScroll);
 	}
+
+	for(const CSkill * skill : CGI->skillh->objects)
+	{
+		for(int level = 1; level <= 3; level++)
+		{
+			int frame = 2 + level + 3 * skill->id;
+			addImageListEntry(frame, "SECSK32", skill->at(level).iconSmall);

```

Comment:
Just a bit to much duplication, please extract `skill->at(level)` to local variable.